<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qCjyxScalarVolumeDisplayWidget</class>
 <widget class="qCjyxWidget" name="qCjyxScalarVolumeDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scalar Volume Display</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="LookupTableLabel">
     <property name="text">
      <string>Lookup Table:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qDMMLColorTableComboBox" name="ColorTableComboBox">
     <property name="showChildNodeTypes">
      <bool>true</bool>
     </property>
     <property name="addEnabled">
      <bool>false</bool>
     </property>
     <property name="removeEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="InterpolateLabel">
     <property name="text">
      <string>Interpolate:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="InterpolateCheckbox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="WindowLevelPresetsLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Window/Level:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="LockWindowLevelButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/Icons/Medium/CjyxUnlock.png</normaloff>:/Icons/Medium/CjyxUnlock.png</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="qDMMLWindowLevelWidget" name="DMMLWindowLevelWidget"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="qDMMLVolumeThresholdWidget" name="DMMLVolumeThresholdWidget"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="ctkCollapsibleGroupBox" name="HistogramGroupBox">
     <property name="title">
      <string>Histogram</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="ctkTransferFunctionView" name="TransferFunctionView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QWidget" name="PresetsWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qDMMLColorTableComboBox</class>
   <extends>qDMMLNodeComboBox</extends>
   <header>qDMMLColorTableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qDMMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qDMMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qDMMLVolumeThresholdWidget</class>
   <extends>QWidget</extends>
   <header>qDMMLVolumeThresholdWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qDMMLWindowLevelWidget</class>
   <extends>QWidget</extends>
   <header>qDMMLWindowLevelWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qCjyxWidget</class>
   <extends>QWidget</extends>
   <header>qCjyxWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkTransferFunctionView</class>
   <extends>QGraphicsView</extends>
   <header>ctkTransferFunctionView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Resources/qCjyxVolumesModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qCjyxScalarVolumeDisplayWidget</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>ColorTableComboBox</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
